components/edit-collective/UpdateBankDetailsForm.js

```diff
@@ -65,24 +62,25 @@ Please note that it will take a few days to process your payment.`,
     return (
       <Flex flexDirection="column">
         <Container as="fieldset" border="none" width={1} py={3}>
-          <Flex flexDirection="row">
-            <Box mb={3}>
+          <Flex flexDirection={['column-reverse', 'row']}>
+            <Box mb={3} flexGrow={1}>
               <StyledTextarea
                 label={intl.formatMessage(this.messages['bankaccount.instructions.label'])}
                 htmlFor="instructions"
-                width={500}
+                width="100%"
                 height={300}
                 onChange={e => this.onChange('instructions', e.target.value)}
-                defaultValue={
-                  get(value, 'instructions') || intl.formatMessage(this.messages['bankaccount.instructions.default'])
-                }
+                defaultValue={value || intl.formatMessage(this.messages['bankaccount.instructions.default'])}
```

Comment:
Doesn't this code generate a warning saying that the variables (`{amount}`, `{OrderId}`) are not set?

Reply:
Nope.

Reply:
No, it doesn't. It just renders it as plain text.